kits/support-to-engineering-bug-bridge/apps/.env.example

```
@@ -0,0 +1,40 @@
+
+# === Embeddings (Hugging Face) ===
+# Required for sentence-transformers/all-MiniLM-L6-v2 (dim 384) to vectorize ticket text.
+# Scope: Requires a free Read token.
+# Data Policy: Hugging Face does not use payloads sent to the Inference API to train their models.
+# This aligns with the Phase 3 PII stance for ticket text.
+HF_TOKEN=hf_your_huggingface_token_here
+
+# === Lamatic ===
+# Required for triggering the flow and writing to the vector store.
+LAMATIC_API_URL=https://api.lamatic.ai/v1
+LAMATIC_API_KEY=lamatic_your_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_API_KEY key should go before the LAMATIC_API_URL key

(UnorderedKey)